conf.py

```diff
@@ -4,50 +4,124 @@
 # Digital Earth Australia documentation build configuration file, created by
 # sphinx-quickstart on Wed Jun  7 17:22:24 2017.
 
+import os
+import sys
+
 extensions = [
-    'nbsphinx',
-    'sphinx.ext.autodoc',
-    'sphinx.ext.autosummary',
-    'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.mathjax',
-    'sphinx.ext.viewcode',
-    'sphinx.ext.todo',
-    'recommonmark',
+    "nbsphinx",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.napoleon",
```

Comment:
Ah, I assume this was also needed, along with the mocking and adding dea_tools path

Reply:
Ah yep, I added `sphinx.ext.napolean` (as previously done by Africa) to correctly interpret the Numpy-style docstrings - without it they get formatted without the nice "Parameters" and "Returns" structure/formatting
https://github.com/digitalearthafrica/deafrica-docs/blob/main/conf.py#L35

Reply:
With: 
![image](https://user-images.githubusercontent.com/17680388/185316624-2f07c45e-f6c4-400d-8bfd-dc13c7129ef8.png)

Without:
![image](https://user-images.githubusercontent.com/17680388/185317156-f3e1057b-3da2-4d78-9d69-2bc548ec6059.png)